app/src/main/res/layout/home_tile.xml

```diff
@@ -1,40 +1,58 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              xmlns:tools="http://schemas.android.com/tools"
-              android:orientation="vertical"
-              android:layout_width="140dp"
-              android:layout_height="100dp"
-              android:background="@drawable/home_tile_background"
-              android:foreground="@drawable/home_tile_selector"
-              android:focusable="true"
-              android:clickable="true"
-              android:layout_marginStart="12dp"
-              android:layout_marginEnd="12dp"
-              android:layout_marginBottom="18dp">
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="140dp"
+    android:layout_height="100dp"
+    android:layout_marginBottom="18dp"
+    android:layout_marginEnd="12dp"
+    android:layout_marginStart="12dp"
+    android:background="@drawable/home_tile_background"
+    android:clickable="true"
+    android:clipChildren="false"
+    android:focusable="true"
+    android:foreground="@drawable/home_tile_selector"
+    android:orientation="vertical">
 
     <ImageView
+        android:id="@+id/home_tile_menu_icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="end|top"
+        android:layout_marginEnd="-15dp"
+        android:layout_marginTop="-15dp"
+        android:background="@drawable/circle_button_background"
+        android:elevation="4dp"
+        android:src="@drawable/ic_home_menu"
```

Comment:
can we use the same icon Jeff has for the overlay - ic_wide_menu
and then tint the icon used for our purpose on the home screen as #2a2a2e